apps/metadata-editor-e2e/src/e2e/dashboard.cy.ts

```diff
@@ -46,6 +37,9 @@ describe('dashboard', () => {
         .then((list) => {
           originalFirstItem = list.trim()
           cy.get('.record-table-header').first().click()
+          // Takes time to refresh results
+          // eslint-disable-next-line cypress/no-unnecessary-waiting
+          cy.wait(500)
```

Comment:
My understanding is that this kind of `wait` should not be necessary because before declaring that an assertion fails, Cypress will "retry" several times for 2 seconds; what happens if you remove it?

Reply:
On my computer it fails one out of five times. I remove to see how it works with gh.